V8ed/Utils/logger.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Vroumed.V8ed.Utils;
+
+public static class Logger
+{
+  // Dictionnaire pour stocker les chemins des fichiers de log
+  private static readonly Dictionary<string, string> _logFiles = new();
+
+  // Méthode pour enregistrer un fichier de log avec un nom spécifique
+  public static void RegisterLogFile(string logName, string filePath)

```

Comment:
With the enum implementation, change signature of RegisterLogFile to have as "logName" the enum entry

and use `Enum.GetValues(typeof(LogFile)).Cast<LogFile>();` to loop through the enums in the constructor, then use a toString() to get the string equivalent of that particular enum